manifests/0000_50_cluster-platform-operator-manager_00-platformoperator.crd.yaml

```diff
@@ -18,100 +18,143 @@ spec:
     singular: platformoperator
   scope: Cluster
   versions:
-    - name: v1alpha1
```

Comment:
Note: This is just due to identification diff between the last recorded version of this PlatformOperators CRD manifest. IIRC, the o/api bumped the yq dependency which may explain the indentation diff.